varsome_api/models/variant.py

```diff
@@ -0,0 +1,125 @@
+from .fields import DictField
+from .elements import *
+
+__author__ = "ckopanos"
+
+
+class AnnotatedVariant(models.Base):
+    """
+    Base variant result definition model
+    Most fields are defined as list fields, even though they contain a single list item
+    This is due to the fact that VarSome API can return both current and old versions of databases
+    """
+
+    chromosome = fields.StringField(help_text="Chromosome")
+    alt = fields.StringField(help_text="ALT Sequence", required=False, nullable=True)
+    ref = fields.StringField(help_text="REF Sequence", required=False, nullable=True)
+    pos = fields.IntField(help_text="Position")
+    variant_id = fields.StringField(help_text='Variant Id')
+    refseq_transcripts = fields.ListField(required=False, items_types=(Transcript,), help_text="RefSeq Transcripts")
+    ensembl_transcripts = fields.ListField(required=False, items_types=(Transcript,), help_text="Ensembl Transcripts")
+    broad_exac = fields.ListField(required=False, items_types=(ExAC,), help_text="ExAC")
+    gnomad_exomes = fields.ListField(required=False, items_types=(GnomAD,), help_text="gnomAD Exomes (ExAC)")
+    gnomad_exomes_coverage = fields.ListField(required=False, items_types=(GnomADCoverage,),
+                                              help_text="gnomAD exomes coverage")
+    gnomad_genomes = fields.ListField(required=False, items_types=(GnomAD,), help_text="gnomAD Genomes")
+    gnomad_genomes_coverage = fields.ListField(required=False, items_types=(GnomADCoverage,),
+                                               help_text="gnomAD genomes coverage")
+    thousand_genomes = fields.ListField(required=False, items_types=(ThousandGenomes,), help_text="1000 Genomes")
+    gerp = fields.ListField(required=False, items_types=(Gerp,), help_text="GERP")
+    isb_kaviar3 = fields.ListField(required=False, items_types=(Kaviar3,), help_text='ISB Kaviar3')
+    dbnsfp = fields.ListField(required=False, items_types=(DbNSFP,), help_text="dbNSFP")
+    dann_snvs = fields.ListField(required=False, items_types=(DannSNVs,), help_text="DANN score")
+    dbnsfp_dbscsnv = fields.ListField(required=False, items_types=(DBscSNV,), help_text='dbNSFP dbscSNV')
+    ncbi_dbsnp = fields.ListField(required=False, items_types=(DbSNP,), help_text="dbSNP")
+    sanger_cosmic = fields.ListField(required=False, items_types=(Cosmic,), help_text="Sanger Cosmic")
+    sanger_cosmic_public = fields.ListField(required=False, items_types=(CosmicPublic,), help_text="Cosmic")
+    sanger_cosmic_licensed = fields.ListField(required=False, items_types=(CosmicLicensed,), help_text="Cosmic")
+    ncbi_clinvar = fields.ListField(required=False, items_types=(ClinVar,), help_text="ClinVar")
```

Comment:
ncbi_clinvar is not in the API anymore

Reply:
Well the schema still returns them, and the available get parameters too. I will remove from the json models. Maybe we should also patch the api and remove the parameters, but we should keep the serialzers for the portal..

Reply:
Ok done, removed clinvar (1). I will prepare a patch for the api as well